content/en/blog/_posts/2023-04-04-in-place-pod-resize/index.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: blog
+title: "Kubernetes 1.27: In-place Resource Resize for Kubernetes Pods (alpha)"
+date: 2023-04-04
```

Comment:
Please change this folder name and the release date tag according to the suggestion.

Reply:
@harshitasao @tengqm Updated, please review. 
thanks.